### ISSUE TITLE
Please, add ReactOS to supported distribution

### ISSUE THREAD

**Issue by petr-akhlamov** [unetbootin/unetbootin#49]
Hello. Please, add to Unetbootin support of ReactOS.
For reactos need syslinux 4.05 and later. ReactOS developer Hermes Belusca made usb-key with syslinux: http://www.youtube.com/watch?v=eancDwmWyCw
For hybrid image, bootcd or livecd, after unpacking, need specify boot driver: /loader/setupldr.sys
I checked it in GRUB, on HDD. https://youtu.be/gtxvVeJO8NY?t=1m32s
Already RuFS supported ReactOS, but your program doesn't format flash drive and cross platformed.

**Comment by ousia** [comment]
:+1: 

That would be a great feature to have.

**Comment by vladgoloyad** [comment]
Nice!

**Comment by yllekz** [comment]
Yes please. Tried it today and unetbootin didn't work with the ReactOS ISO.
